http-gateway/web/src/containers/things/things-details-page.js

```diff
@@ -48,17 +56,78 @@ export const ThingsDetailsPage = props => {
     breadcrumbs.push({ label: deviceName })
   }
 
+  const fetchResourceAndOpenModal = async ({
+    di,
+    href,
+    currentInterface = '',
+  }) => {
+    // If there is already a fetch for a resource, disable the next attempt for a fetch untill the previous fetch finishes
+    if (loadingResource) {
+      return
+    }
+
+    setLoadingResource(true)
+
+    try {
+      const interfaceGetParam = currentInterface
+        ? `interface=${currentInterface}`
+        : ''
+      const {
+        data: { if: ifs, rt, ...resourceData }, // exclude the if and rt

```

Comment:
in this case I would give it a better name than ifs.

Reply:
Looks like ifs is not even used though

Reply:
This is this parameters official name. The name change I did here from "if" to "ifs" is only because `if` is a reserved keyword for conditions.

Reply:
Yes, it is not used, not the `rt`, as the comment at the end of line states, it is only for excluding it from the object.